integrity-backend/input-browsertrix-examples/3e11cc57daf3bad8375935cad4878123acc8d769551ff90f1b1bb0dc597-meta-content.json

```diff
@@ -48,7 +48,7 @@
               "behaviors": "autoscroll,autoplay,autofetch,siteSpecific"
             }
           },
-          "additional_data":
+          "additionalData":
```

Comment:
Updated to camel case, because this is defined and keeping in formatting of rest of json